<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookView</class>
 <widget class="QWidget" name="AddBookView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="viewBackground" stretch="0,2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="windowTitleLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#titleLabel{
	
	color: rgb(0, 0, 0);
	background-color: rgb(215, 236, 199);
}</string>
       </property>
       <property name="text">
        <string>    Add New Book</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="bookInfoLayout">
       <item>
        <layout class="QGridLayout" name="inputsLayout">
         <item row="2" column="1">
          <widget class="QLineEdit" name="isbnLineEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="descriptionLabel">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="genreLineEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="titleLabel">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="authorLabel">
           <property name="text">
            <string>Author</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="isbnLabel">
           <property name="text">
            <string>ISBN</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QTextEdit" name="descriptionTextEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="titleLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="authorLineEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="sectionLabel">
           <property name="text">
            <string>Section</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="genreLabel">
           <property name="text">
            <string>Genre</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="sectionLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Add Book</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
